packages/nodes-base/nodes/Todoist/v2/OperationHandler.ts

```diff
@@ -133,7 +256,8 @@ export class DeleteHandler implements OperationHandler {
 
 export class GetHandler implements OperationHandler {
 	async handleOperation(ctx: Context, itemIndex: number): Promise<TodoistResponse> {
-		const id = ctx.getNodeParameter('taskId', itemIndex) as string;
+		const id = ctx.getNodeParameter('taskId', itemIndex);
+		assertIsStringOrNumber('taskId', id);
```

Comment:
maybe these could be merged into its own function, but I have decided this is out of scope here.